packages/surveys/src/components/questions/OpenTextQuestion.tsx

```diff
@@ -109,6 +110,8 @@ export const OpenTextQuestion = ({
                 className="fb-border-border placeholder:fb-text-placeholder fb-text-subheading focus:fb-border-brand fb-bg-input-bg fb-rounded-custom fb-block fb-w-full fb-border fb-p-2 fb-shadow-sm focus:fb-outline-none focus:fb-ring-0 sm:fb-text-sm"
                 pattern={question.inputType === "phone" ? "[0-9+ ]+" : ".*"}
                 title={question.inputType === "phone" ? "Enter a valid phone number" : undefined}
+                minlength={question.inputType === "text" ? question.minLength || undefined : undefined}
+                maxlength={question.inputType === "text" ? question.maxLength || undefined : undefined}
```

Comment:
1. The reason I did not went for camelCase was because there was this weird error where minLength and maxLength was converting `undefined` to `0`. This was not letting me type anything at all. That is why went ahead with lowercase attributes.
2. Added visual feedback about char limits.
3. Limits were required only for text based input type.

Reply:
Regarding the first point, I already applied the changes you suggested, but it still didn't worked. It seems undefined value is getting converted to 0 in the dom.